README.md

```diff
@@ -1,3 +1,3 @@
-# sml
+# Faculty CLI
 
-`sml` is the command line interface to [SherlockML](https://sherlockml.com).
+`faculty` is the command line interface to the [Faculty](https://sherlockml.com). # TODO change
```

Comment:
Markdown uses HTML comment syntax: ` <!-- TODO Change link destination -->`. Also:
```suggestion
`faculty` is the command line interface to [Faculty](https://sherlockml.com). # TODO change
```

Reply:
Could also say 'the Faculty platform'.